Adyen/Client.cs

```diff
@@ -53,7 +54,12 @@ public Client(string username, string password, Environment environment, string
                 Environment = environment
             };
             SetEnvironment(environment, liveEndpointUrlPrefix);
-            _client = new HttpClientWrapper(Config, new System.Net.Http.HttpClient());
+            
+            // Set Timeout for HttpClient
+            var httpClient = new System.Net.Http.HttpClient();
+            httpClient.Timeout = Config.Timeout == default ? TimeSpan.FromMilliseconds(100000) : TimeSpan.FromMilliseconds(Config.Timeout);

```

Comment:
Let's assign a timeout only if the user explicitly provides one. 
```suggestion
            if (Config.Timeout != default)
            {
                httpClient.Timeout = TimeSpan.FromMilliseconds(Config.Timeout);
            }

```

Reply:
Really wanted to use a one liner here and the default is 100000 milliseconds haha